test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -798,6 +798,109 @@ def test_remote_primary_multinode(self):
         except KeyError:
             self.fail("mom_priv/config not found in %s's snapshot" % host2)
 
+    def test_snapshot_obf_stress(self):
+        """
+        A stress test to make sure that snapshot --obufscate really obfuscates
+        the attributes that it claims to
+        """
+        real_values = {}
+
+        # We will try to set all attributes which --obfuscate anonymizes
+        manager1 = str(MGR_USER) + '@*'
+        self.server.manager(MGR_CMD_SET, SERVER,
+                            {ATTR_managers: (INCR, manager1)},
+                            sudo=True)
+        manager2 = str(TEST_USER) + "@*"
+        self.server.manager(MGR_CMD_SET, SERVER,
+                            {ATTR_managers: (INCR, manager2)},
+                            sudo=True)
+        real_values[ATTR_managers] = [manager1, manager2]
+
+        operator = str(OPER_USER) + '@*'
+        self.server.manager(MGR_CMD_SET, SERVER,
+                            {ATTR_operators: (INCR, operator)},
+                            sudo=True)
+        real_values[ATTR_operators] = [operator]
+
+        real_values[ATTR_SvrHost] = [self.server.hostname]
+
+        # Create a queue with acls set
+        a = {ATTR_qtype: 'execution', ATTR_start: 't', ATTR_enable: 't',
+             ATTR_aclgren: 't', ATTR_aclgroup: TSTGRP0,
+             ATTR_acluser: TEST_USER}
```

Comment:
I think you need to str() TSTGRP0 and TEST_USER

Reply:
manager() does it already, that's why some tests pass in int values when setting ncpus on vnodes.